.codecov.yaml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "*-agora-submodules-*"
```

Comment:
I think this should be just `submodules-*.lst`? The files seem to start with `submodules` from what I can tell.

Reply:
E.g.:

```
-rw-------    1 andrejmitrovic  staff   3.2K Oct 24 18:20 source-scpd-types-Utils.lst
-rw-------    1 andrejmitrovic  staff   3.6K Oct 24 18:20 source-scpd-types-XDRBase.lst
-rw-------    1 andrejmitrovic  staff   2.3K Oct 24 18:20 submodules-vibe-core-source-vibe-appmain.lst
-rw-------    1 andrejmitrovic  staff   2.0K Oct 24 18:20 submodules-ocean-src-ocean-LibFeatures.lst
-rw-------    1 andrejmitrovic  staff   5.2K Oct 24 18:20 submodules-ocean-src-ocean-core-Array.lst
```